libpod/options.go

```diff
@@ -280,6 +280,18 @@ func WithPrivileged(privileged bool) CtrCreateOption {
 	}
 }
 
+// WithNoNewPrivs sets the noNewPrivs flag in the container runtime
+func WithNoNewPrivs(noNewPrivs bool) CtrCreateOption {
```

Comment:
Do we need this? It occurs to me that it's already in the spec, so we can get at the bool there

Reply:
It needs to show up in `podman inspect` though.

Reply:
Hm. We can probably fix that on the `libpod` side, but that can happen later. Now that we're backed by boltdb removing DB fields is no longer that big of a deal.